### PR TITLE
Add intellij guidance

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
Based on guidance here (https://github.com/errata-ai/vale-server/issues/39), I added a stub of a page for using Vale with IntelliJ

### CODE REVIEW

File: website/docs/vale-server/clients/intellij.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+* * *
```
Comment (nschonni) [inline website/docs/vale-server/clients/intellij.md:1]: Think this needs to be done as YAML frontmatter like
```
---
id: google-docs
title: Google Docs
---
```

Comment (ChrisChinchilla) [reply website/docs/vale-server/clients/intellij.md]: Thanks @nschonni I have a markdown linter that seems to break front matter sometimes :(

**Comment by ChrisChinchilla** [comment]
Added some more input @jdkato

**Comment by jdkato** [comment]
This looks good to me -- Is it ready to be merged?

**Comment by ChrisChinchilla** [comment]
@jdkato Yup!

**Pull request merged**